core/lib/presentation/views/html_viewer/html_content_viewer_on_web_widget.dart

```diff
@@ -69,7 +69,17 @@ class _HtmlContentViewerOnWebState extends State<HtmlContentViewerOnWeb> {
       <script type="text/javascript">
         window.parent.addEventListener('message', handleMessage, false);
         window.addEventListener('click', handleOnClickLink, true);
-      
+        window.onload = function() {
+          addTargetBlank();
+        };
+        
+        function addTargetBlank() {
```

Comment:
IMO, you can handle the `a` tag in `HtmlTransform` after getting the email body from the server. To avoid long loading for `web` when displaying.

Reply:
@dab246 package html flutter not support setAttribute. Therefore I used js to setAttribute `a` tag.

Reply:
Use package: `package:html/dom.dart`. Ex:
```dart
final elements = document.querySelectorAll('a');
await Future.wait(elements.map((element) async {
  element.attributes['target'] = '_blank';
});
```